File: autox/src/autox/actions/Start.java
package autox.actions;

/**
 * Created with AutoX project.
 * User: jien.huang
 * Date: 12/15/12
 *
 * <Step Action="autox.actions.Start" />
 */
public class Start extends Action {
    @Override
    public Object findTestObject() {
        return null;
    }

    @Override
    protected void handle(Object testObject) {

        BrowserManager.getInstance().start();

    }
}

File: dotnet/AutoX.Client/Start.cs
﻿// Hapa Project, CC
// Created @2012 08 24 09:25
// Last Updated  by Huang, Jien @2012 08 24 09:25

#region

using System.Xml.Linq;

#endregion

namespace AutoX.Client
{
    public class Start : AbstractAction
    {
        public override XElement Act()
        {
            var sr = new StepResult(this);
            Browser.GetInstance().GetCurrentBrowser();
            return sr.GetResult();
        }
    }
}